grcs/0020-knowledge-graph.md

```diff
@@ -373,100 +358,105 @@ An op represents a single atomic operation that produces or modifies knowledge.
 ```proto
 message Op {
   OpType type = 1;
-  Triple triple = 2;
-  Entity entity = 3;
-  Relation relation = 4;
-  repeated Triple triples = 5;
+  Entity entity = 2;
+  Relation relation = 3;
+  Property property = 4;
 }
-```
+``` 
 
 ```proto
 message Relation {
-  string id = 1;
-  string type = 2;
-  string from_entity = 3;
-  string to_entity = 4;
-  string index = 5;
+  bytes id = 1;
+  bytes type = 2;
+  bytes from_entity = 3;
+  bytes from_property = 4;
+  bytes to_entity = 5;
+  bytes to_property = 6;
+  bytes to_space = 7;
+  bytes entity = 8;
+  bool verified = 9;
```

Comment:
what does `verified` mean and what's the purpose of setting it when creating a relation?